neptune/experiments.py

```diff
@@ -359,6 +374,12 @@ def send_graph(self, graph_id, value):
 
         self._client.put_tensorflow_graph(self, graph_id, value)
 
+    def reset_log(self, log_name):
+        channel = self._find_channel(log_name, ChannelNamespace.USER)
+        if channel is None:
+            raise ChannelDoesNotExist(self.id, log_name)
+        self._client.reset_channel(channel.id)
```

Comment:
You can add a try-catch block to change exception to a more user-friendly exception when backend does not resolve channel id

Reply:
It should never be a case since this id was just fetched from backend.